docs/core/rid-catalog.md

```diff
@@ -142,6 +151,12 @@ See [Prerequisites for .NET Core on Windows](windows-prerequisites.md) for more
   - `rhel.7.2-x64`
   - `rhel.7.3-x64` (.NET Core 2.0 or later versions)
   - `rhel.7.4-x64` (.NET Core 2.0 or later versions)
+- SUSE Enterprise Linux (SLES) (.NET Core 2.0 or later versions)
+  - `sles-x64`
+  - `sles.12-x64`
```

Comment:
Listing all the supported versions here is not sustainable. It will be always out of date and incomplete here. It duplicates product support matrix from the linked doc.

I would trim RIDs for specific versions from here. Only keep them where the RID for specific version really matters (i.e. RedHat 6).

Reply:
I'm not sure which RIDs really matter and which don't. Is there a resource for me to reference to make that call?

Reply:
The stock .NET Core is built for the following RIDs:
- Windows 
- OSX
- Linux (portable)
- Linux Musl
- Linux RedHat 6 (rhel.6)

Most Linux distros use glibc as C runtime. The portable Linux build works for all of them. It means that you rarely need to worry about whether you are running on Ubuntu, SUSE, Fedora, etc. You can just use portable Linux RID and it works fine.

The lightweight Linux distros (Alpine, Void, ...) use musl as C runtime: https://wiki.musl-libc.org/projects-using-musl.html. The one Linux-musl .NET Core builds works for all of them.

RedHat 6 is an exception. It is very old and required many workarounds that it was not practical to fold into portable Linux build. It got its own special build.

If you use the distro-specific RID, it will get mapped to one of 3 above RIDs. We have been de-emphasizing the distro-specific RIDs for this reason and we have not bothered to add RIDs for many of the less common Linux distros out there. The above 3 Linux RIDs are the ones that matter.

Reply:
I pushed a new version that simplified the Linux section and added a note to say portable RIDs are preferred when targeting .NET Core 2.0 and higher to the line that was already in the document about portable RIDs being added in .NET Core 2.0.

Reply:
Your latest update looks good to me, modulo one nit. Thank you!